test/suggestion/v1beta1/test_skopt_service.py

```diff
@@ -187,6 +187,117 @@ def test_get_suggestion(self):
+                    api_pb2.AlgorithmSetting(name="unknown_conf", value="1111")]
+            ))
+        _, _, code, details = call_validate()
+        self.assertEqual(code, grpc.StatusCode.INVALID_ARGUMENT)
+        self.assertEqual(details, 'unknown setting unknown_conf for algorithm bayesianoptimization')
+
+        # unknown base_estimator
+        experiment_spec[0] = api_pb2.ExperimentSpec(
+            algorithm=api_pb2.AlgorithmSpec(
+                algorithm_name="bayesianoptimization",
+                algorithm_settings=[
+                    api_pb2.AlgorithmSetting(name="base_estimator", value="unknown estimator")]
+            ))
+        _, _, code, details = call_validate()
+        wrong_algorithm_setting = experiment_spec[0].algorithm.algorithm_settings[0]
+        self.assertEqual(code, grpc.StatusCode.INVALID_ARGUMENT)
+        self.assertEqual(details,
+                         f'{wrong_algorithm_setting.name} {wrong_algorithm_setting.value} is not supported in katib')
+
+        # wrong n_initial_points
+        experiment_spec[0] = api_pb2.ExperimentSpec(
+            algorithm=api_pb2.AlgorithmSpec(
+                algorithm_name="bayesianoptimization",
+                algorithm_settings=[
+                    api_pb2.AlgorithmSetting(name="n_initial_points", value="-1")]
+            ))
+        _, _, code, details = call_validate()
+        wrong_algorithm_setting = experiment_spec[0].algorithm.algorithm_settings[0]
+        self.assertEqual(code, grpc.StatusCode.INVALID_ARGUMENT)
+        self.assertEqual(details, f'{wrong_algorithm_setting.name} should be great or equal than zero')
+
+        # unknown acq_func
+        experiment_spec[0] = api_pb2.ExperimentSpec(
+            algorithm=api_pb2.AlgorithmSpec(
+                algorithm_name="bayesianoptimization",
+                algorithm_settings=[
+                    api_pb2.AlgorithmSetting(name="acq_func", value="unknown")]
+            ))
+        _, _, code, details = call_validate()
+        wrong_algorithm_setting = experiment_spec[0].algorithm.algorithm_settings[0]
+        self.assertEqual(code, grpc.StatusCode.INVALID_ARGUMENT)
+        self.assertEqual(details,
+                         f'{wrong_algorithm_setting.name} {wrong_algorithm_setting.value} is not supported in katib')
+
+        # unknown acq_optimizer
+        experiment_spec[0] = api_pb2.ExperimentSpec(
+            algorithm=api_pb2.AlgorithmSpec(
+                algorithm_name="bayesianoptimization",
+                algorithm_settings=[
+                    api_pb2.AlgorithmSetting(name="acq_optimizer", value="unknown")]
+            ))
+        _, _, code, details = call_validate()
+        wrong_algorithm_setting = experiment_spec[0].algorithm.algorithm_settings[0]
+        self.assertEqual(code, grpc.StatusCode.INVALID_ARGUMENT)
+        self.assertEqual(details,
+                         f'{wrong_algorithm_setting.name} {wrong_algorithm_setting.value} is not supported in katib')
```

Comment:
The same comment about message format style.

Reply:
Done